.github/workflows/build.yml

```
@@ -0,0 +1,49 @@
+name: Build_docker.yml
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+    outputs:
+      python-version: $({matrix.python-version})
```

We do not need this variable. Instead define `app-version` here.
```suggestion
      app-version: ["0.1.1"]
```